config.js

```diff
@@ -375,6 +375,20 @@ var config = {
         // shard: "shard1",
         // region: "europe",
         // userRegion: "asia"
+    },
+
+    // Information for the chrome extension banner
+    chromeExtensionBanner: {
+        // The chrome extension to be installed address
+        url: 'https://chrome.google.com/webstore/detail/jitsi-meetings/kglhbbefdnlheedjiejgomgmfplipfeb',
```

Comment:
This chrome extension should not go in config ... People that install jitsi-meet should configure their own extension that is pointed to their own deployment.

Reply:
The ask was to check whether the jitsi extension is installed for jitsi-meet , or if the 2 8x8 chrome extensions are installed for 8x8-jitsi-meet,  so some static values.

The case in which users install jitsi-meet do not apply in the web browser  case when users access a meeting direclty for example, but the banner should still be presented in this case.

So this is not thought out for being exposed to jitsi-meet integrators, but to be shown for end-users in order to advertise our own extensions

Reply:
First of all, you are putting deployment-specific information in an opensource project which doesn't make sense.
I'll give you an example: I'm company X and I have a domain X.net. I install jitsi-meet cause I want to use it and suddenly it starts offering me and extension, I install it, in the case above it is an extension that is for meet.jit.si deployment and suddenly I start seeing an extension in my Calendar and start scheduling meetings which will use https://meet.jit.si deployment and not X.net.
If you go to https://meet.jit.si/config.js you will see `desktopSharingChromeExtId: 'kglhbbef....` config, do you see that same id here in config.js? No, cause this is deployment-specific setting in doesn't make sense to be added in the source code. 
So please remove this setting leaving a commented example what's need to be added.

Reply:
Oh ok I see what you mean, in the opensource project I'll leave it as is but commented as an example, and add the pre-environment configuration in the deployment repository per-environment. I initially understood that you mean that it should be exposed on the jitsi api. Sorry for the confusion.

Reply:
commented out the config